docs/manual/java/guide/devmode/ConfiguringServicesInDevelopment.md

```diff
@@ -1,3 +1,21 @@
+# How are addresses bound by services?
+
+By default, Lagom services bind to `localhost`. This address can be changed with the following.
+
+```xml
```

Comment:
Add before this: "In Maven, you can do this by modifying the service implementation's pom configuration:"

(like in the "How are ports assigned to services?" section below… which is missing the apostrophe currently and would be nice to fix up too)